src/components/Buttons/IconButton/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { Fab as MaterialFab, FabProps as MaterialFabProps } from "@material-ui/core";
+
+export interface FabBaseProps extends MaterialFabProps {}
+
+const IconButton = (props: FabBaseProps) => {
```

Comment:
```suggestion
const IconButtonBase = (props: FabBaseProps) => {
```
To prevent mistaking it with the material ui's IconButton